### PR TITLE
Add styles and create responsive pages

### PR DESCRIPTION
**Pull request by upsu90**
- hamburger menu
- background for all devices
- fixed footer
- some global styles
- Login pages
- Signup pages
- Scientific works
- My reviews
- My work / Scientific work page (without versions)

### CODE REVIEW

File: web-client/src/components/Account/Avatar.jsx [hunk 1]
```diff
@@ -1,83 +1,109 @@
...
+  const [openAlertError, SetOpenAlertError] = React.useState(false);
+  const durationOfAlert = 4000;
+
+  // Show alert
+  const ShowAlert = () => {
+    SetOpenAlertError(true);
+  };
+
+  // Close alert
+  const CloseAlert = (_, reason) => {
```
Comment (JustD4nTe) [inline web-client/src/components/Account/Avatar.jsx]: Pointless comments

File: web-client/src/components/Navigation.jsx [hunk 2]
```diff
@@ -73,29 +92,79 @@ export default function Navigation(props) {
...
+    },
   })();
 
   const Logout = () => {
     localStorage.removeItem("jwt");
     window.location.href = URL;
   };
 
-  const path = `${LINKS.WORKS}/${props.userInfo.scientificWorkId}`;
+  const [open, setOpen] = React.useState(false);
```
Comment (JustD4nTe) [inline web-client/src/components/Navigation.jsx]: The name doesn't say what it is. You should change it to be more meaningfully

File: web-client/src/components/PopUpForgotPassword.jsx [hunk 3]
```diff
@@ -107,15 +117,14 @@ export default function PopUpForgotPassword(props) {
...
-        onClick={handleClickOpen}
-      >
+      <Button className={style.textButton} color="primary" onClick={handleClickOpen}>
         Forgot password?
       </Button>
 
       <Dialog open={open} onClose={handleClose}>
+      <div className={style.divClose}>
+            <Close className={style.close} onClick={handleClose} />
+          </div>
```
Comment (JustD4nTe) [inline web-client/src/components/PopUpForgotPassword.jsx]: rly?

File: web-client/src/components/ScientificWorkList/OneWork.jsx [hunk 4]
```diff
@@ -97,10 +117,18 @@ export default function OneWork(props) {
...
+  const showStatus = (
+    <Button variant="contained" className={style.btnStatus}>
+      {props.status}
+    </Button>
+  );
+  //https://www.xspdf.com/resolution/50694881.html <- information for button download
   return (
     <div className={style.main}>
-      <h2 className={style.h2}>{props.title}</h2>
+      <div className={style.status}>{props.status ? showStatus : null}</div>
```
Comment (JustD4nTe) [inline web-client/src/components/ScientificWorkList/OneWork.jsx]:
```suggestion
      <div className={style.status}>{props.status && showStatus}</div>
```

File: web-client/src/pages/ScientificWorks/scientific-works.js [hunk 5]
```diff
@@ -88,62 +90,64 @@ export default function ScientificWorks() {
...
+    <RecentWorks path={recent.path} name={recent.name} description={recent.description} />
   ));
 
   return (
     <div className={style.main}>
       <h1>Scientific works</h1>
+      <div className={style.works}>
+        <div className={style.list}>
+          {/* If list of works is null, then nothing is displayed */}
+          {works[0]?.title ? workList : null}
```
Comment (JustD4nTe) [inline web-client/src/pages/ScientificWorks/scientific-works.js]:
```suggestion
          {works[0]?.title && workList}
```

**Review by JustD4nTe — CHANGES_REQUESTED** [review]
image
These buttons in hamburger menu are too close

image
image
sign up (mobile view)

File: web-client/src/components/Account/SignInForm.jsx [hunk 6]
```diff
@@ -141,33 +170,50 @@ export default function SignInForm(props) {
...
+                style={messageStyle}
+                className={style.formHelperText}
+              >
+                {"Error: Incorrect password or/and email"}
+              </FormHelperText>
+            </FormControl>
+            <div className={style.loginLinks}>
               {/* Forgot password */}
               <PopUpForgotPassword SetEmail={passEmail} />
               {console.log(forgotEmail)}
```
Comment (Kwolik) [inline web-client/src/components/Account/SignInForm.jsx]: @upsu90 pls remove. I I forgot xd

**Review by Kwolik — COMMENTED** [review]
image


Mobile version popup Forgot password.
image

Why is there a big gap between the job information and the job versions?
image


larger view of the PDF?
image

File: web-client/src/components/Account/SignInForm.jsx (hunk 6 above)
Comment (JustD4nTe) [reply web-client/src/components/Account/SignInForm.jsx]: Do you wanna remove these 7 lines?

Comment (Kwolik) [reply web-client/src/components/Account/SignInForm.jsx]: only console.log

**Comment by upsu90** [comment]
@Kwolik 
> image
Can you tell me more? it works on my machine :p

> Mobile version popup Forgot password.
> image
I know, it's not for this PR (I'll change it later)

> Why is there a big gap between the job information and the job versions?
> image
It's for other task

> larger view of the PDF?
> image
Unfortunately, it's very complicated to change it so we have to leave it for a while

**Review by KamilDonda — APPROVED** [review]

**Review by JustD4nTe — APPROVED** [review]

**Review by Kwolik — APPROVED** [review]

**Pull request merged**